scipy/ndimage/_morphology.py

```diff
@@ -202,6 +202,11 @@ def generate_binary_structure(rank, connectivity):
             [False,  True, False],
             [False, False, False]]], dtype=bool)
 
+    This is a viusal presentation of `generate_binary_structure` in 3D:
+
+    .. plot:: tutorial/examples/ndimage/3D_binary_structure.py
```

Comment:
Seems like we cannot use `plot` in the docstrings (CI). Sorry about that, I never tried. Either we put this in the tutorial or inline (fine since the snippet is short).